src/ucp/core/ucp_request.h

```diff
@@ -199,7 +201,20 @@ typedef struct ucp_recv_desc {
 } ucp_recv_desc_t;
 
 
+/**
+ * Multirail rendezvous-get info.
+ */
+typedef struct ucp_rndv_get_mrail {
+    ucp_lane_index_t      rail_idx;
+    struct {
+        ucp_lane_index_t  lane;
+        uct_rkey_bundle_t rkey_bundle;
+    } rail[UCP_MAX_RAILS];
```

Comment:
why not use UCP_MAX_LANES instead of UCP_MAX_RAILS in all places?

Reply:
there is limitations for structure size, and 4 rails is maximum value to avoid structure size increase